ice.c

```diff
@@ -3050,23 +3047,19 @@ static void janus_ice_cb_nice_recv(NiceAgent *agent, guint stream_id, guint comp
 							p->last_retransmit = now;
 							retransmits_cnt++;
 							/* Enqueue it */
-							janus_ice_queued_packet *pkt = g_malloc(sizeof(janus_ice_queued_packet));
-							pkt->data = g_malloc(p->length+SRTP_MAX_TAG_LEN);
+							janus_ice_queued_packet *pkt = g_malloc0(sizeof(janus_ice_queued_packet));
+							pkt->data = g_malloc0(p->length+SRTP_MAX_TAG_LEN);
 							memcpy(pkt->data, p->data, p->length);
```

Comment:
In this instance `pkt->data` is immediately written to so I think `g_malloc0` is not needed/worth it.